webapp/ordering/models.py

```diff
@@ -257,8 +257,12 @@ def send_ride_mails(self):
 
         rides = self.rides.all()
         for ride in rides:
-            drivers = [ride.driver, ride.codriver]
-            for user in drivers:
+            involved = [
+                ride.driver,
+                ride.codriver,
+                self.distribution_coordinator
```

Comment:
Can be `None` and then it will break, maybe do:
```
involved = [
    ride.driver,
    ride.codriver,
]
if self.distribution_coordinator:
    involved.append(self.distribution_coordinator)

```

Reply:
Good catch! 
I've added with a slightly different solution, just in case more of those `involved` users become optional.